src/familybot/web/templates/logs.html

```
@@ -3,478 +3,239 @@
+                        <div style="flex:1;min-width:80px">
+                            <label class="form-label">Limit</label>
+                            <select id="log-limit" class="form-select">
+                                <option value="50">50</option>
+                                <option value="100" selected>100</option>
+                                <option value="250">250</option>
+                                <option value="500">500</option>
+                            </select>
+                        </div>
+                        <div style="flex:2;min-width:180px">
+                            <label class="form-label">Search</label>
+                            <input type="text" id="search-filter" class="form-control" placeholder="Filter log messages…">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)